.eslintrc.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
     'prettier/prettier': 2,
     'import/prefer-default-export': 0,
     'import/no-default-export': 2,
+    "react/jsx-props-no-spreading": "off",
```

Comment:
I don't think we want to turn this off for the _whole_ project, we just want to use spreading to make writing the tests easier.

Can we instead turn off this rule for just the test file?

Also, let's use single quotes here.